docs/how-tos/review-a-request.md

```diff
@@ -29,8 +29,8 @@ It also shows a log of recent activity.
 
 This page also contains buttons for various actions that can be performed
 on the request as a whole (submitting your review, returning the request to
-the researcher etc.).  In the example below, all buttons are disabled, as
-this request has not yet been reviewed.
+the researcher etc.).  In the example below, all buttons (except "Return request")
+are disabled, as this request has not yet been reviewed.
```

Comment:
Is it worth a comment somewhere in the for-researcher's docs on how to create/submit a request to say that if you've made a mistake, you can ask an output-checker to return the request to you in order to make further changes?

Reply:
That's a very good idea.